superset/utils/encrypt.py

```diff
@@ -22,7 +22,12 @@
 from flask_babel import lazy_gettext as _
 from sqlalchemy import text, TypeDecorator
 from sqlalchemy.engine import Connection, Dialect, Row
-from sqlalchemy_utils import EncryptedType
+from sqlalchemy_utils import EncryptedType as SqlaEncryptedType
+
+
+class EncryptedType(SqlaEncryptedType):
```

Comment:
Why is this needed?

Reply:
oh! finally took on removing WARNING log message that became apparent while I paid attention to the boot sequence's STDOUT